condor_watch_q.py

```diff
@@ -1127,7 +1187,10 @@ def make_table(
         for original_row, processed_row in zip(rows, processed_rows)
     ]
 
-    remaining_columns = terminal_columns - len(strip_ansi(lines[0]))
+    try:
+        remaining_columns = terminal_columns - len(strip_ansi(lines[0]))
+    except IndexError:
+        remaining_columns = 0
```

Comment:
I don't quite follow this... if we have no rows, shouldn't `remaining_columns = terminal_columns`?